### DISCUSSION TITLE
pyodide v0.21.2 How do I use wordcloud?

### DISCUSSION THREAD

**Discussion by Caisfan** [pyodide/pyodide#3886] [Q&A]
pyodide v0.21.2 How do I use wordcloud?
ValueError: 'wordcloud-1.8.2.2-pp38-pypy38_pp73-win_amd64.whl' is not a pure Python 3 wheel

**Comment by ryanking13** [comment]
The wordcloud package has been included in Pyodide since 0.22.0, so you will need to upgrade to >= 0.22.0. (We recommend using 0.22.1).

Note that with Pyodide 0.23, wordcloud is not currently available in 0.23 because Pyodide upgraded its Python version to 3.11 and disabled the wordcloud package, which did not support Python 3.11 at the time.

**Reply by Caisfan** [reply]
In version 0.21.2, I use wheel packets, such as :await  micropip. install (' myproject/packages/openpyxl - 3.0.9 - py2. Py3 - none - any. WHL '), use the version found 0.22.1 wordcloud packages directory is the folder, how should I use it

**Reply by Caisfan** [reply]
await pyodide.loadPackage('pandas');
        await pyodide.loadPackage('micropip')
        await pyodide.loadPackage('xlrd')
        await pyodide.loadPackage('matplotlib')
        await pyodide.loadPackage('statsmodels')
        await pyodide.loadPackage('wordcloud')
        await pyodide.runPythonAsync(`
            import wrodcloud
            .........
`)
This is my JavaScript code, Error: No known package with name 'wordcloud' when executed to 'await pyodide.loadPackage('wordcloud')'

**Reply by ryanking13** [reply]
Well, pyodide.loadPackage accepts a URL so you can try installing a wordcloud wheel from Pyodide 0.22.1 by passing a URL, e.g. `pyodide.loadPackage("https://..../wordcloud_.....whl"`. But I can't guarantee that the wheel will be compatible.
